recipes/txmongo/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
You can delete this license file since it is included in the source distribution